src/app/components/OptimoPromos/Wrappers.styles.jsx

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '#app/legacy/gel-foundations/src/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_BORDER,
+  GEL_SPACING_DBL,
+} from '#app/legacy/gel-foundations/src/spacings';
+import { C_WHITE } from '#app/legacy/psammead-styles/src/colours';
+
+export const ContentWrapper = styled.div`
+  padding: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL};
+  }
+`;
+
+export const PromoWrapper = styled.div`
+  position: relative;
+  height: 100%;
+  background-color: ${C_WHITE};
+`;
+
+export const BorderWrapper = styled.div`
```

Comment:
This is a special case, This wrapper only takes care of the border. You would remove the entire component if UX decided to remove the border. I believe the naming convention, in this case, is fine.

I cannot add the border style anywhere that makes it reusable hence this specialised component. Open to suggestions.

Reply:
Why can't the border be in the `PromoWrapper` styles?

If it can't, for things like this I either go with e.g. `PromoOuterWrapper` and `PromoInnerWrapper` however an even better approach would be not having to create an element/component to add a decorative feature and use a CSS pseudo element instead. Give me a shout if you need an explanation

Reply:
It is funny because there was a reason, but I can't remember which one it was 😂, my brain is getting overloaded. It seems to work... so I will change it for now, and if it breaks... well I found my reason.